lib/rules/jsx-no-literals.js

```diff
@@ -55,6 +59,9 @@ module.exports = {
     }
 
     function getValidation(node) {
+      if (allowedStrings && allowedStrings.indexOf(node.value) > -1) {
```

Comment:
should we make a `new Set` of it on line 40?

Reply:
Sure, that would be slightly cleaner